Fix/knockdown

ghostant-1017 opened the pull request.

ghostant-1017 committed:
fix accuracy

ghostant-1017 committed:
fix knockdown if else

ghostant-1017 committed:
add claim transition

jinqiao0423 commented on src/main.leo:
```diff
@@ -59,6 +59,38 @@ program privx_exchange.aleo {
     //     };
     // }
 
```
Should there be a function calling claim_credits, claim_token and claim_token2 all in one?

privx-xyz-lab replied:
Should there be a function calling claim_credits, claim_token, and claim_token2 all in one?

ghostant-1017 committed:
claim 10^8 tokens for each claim 10^8 tokens for each claim 10^8 tokens
for each claim 10^8 tokens for each claim 10^8 tokens for each claim
10^8 tokens for each claim 10^8 tokens for each claim 10^8 tokens for
each claim 10^8 tokens for each claim 10^8 tokens for each
